### PR TITLE
Update guide/command to get or install bbolt

### PR DESCRIPTION
**Pull request by ahrtr**
Fix https://github.com/etcd-io/bbolt/issues/332

Signed-off-by: Benjamin Wang <wachao@vmware.com>

### CODE REVIEW

**Comment by ahrtr** [comment]
The failure is resolved in https://github.com/etcd-io/bbolt/pull/367

File: README.md [hunk 1]
```diff
@@ -78,14 +78,23 @@ New minor versions may add additional features to the API.
...
 ```sh
-$ go get go.etcd.io/bbolt/...
+$ go get go.etcd.io/bbolt@latest
 ```
 
-This will retrieve the library and install the `bolt` command line utility into
-your `$GOBIN` path.
+This will retrieve the library and update your go.mod and go.sum files. 
+
+To run the command line utility, execute:
```
Comment (ptabor) [inline README.md:87]: [sorry] I thought 'edit' will add this as a suggestion, not as a commit on top. But please take a look.

**Review by ptabor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ahrtr) [reply README.md:87]: Looks good to me. Please signoff the commit.

Comment (ptabor) [reply README.md:87]: I'm afraid I cannot force push to your branch to fix it... I only can propose a new commit.

Please squash it or remove my commit. 

I'm sorry for the trouble.

Comment (ahrtr) [reply README.md:87]: NO problem. Let me take care of it.

Comment (ahrtr) [reply README.md:87]: resolved.

**Review by ptabor — APPROVED** [review]
Thank you.

**Pull request merged**